llvm/test/Analysis/UniformityAnalysis/AMDGPU/always_uniform.ll

```diff
@@ -58,6 +67,18 @@ define void @single_lane_func_arguments(i32 %i32, i1 %i1) #2 {
  ret void
 }
 
+; CHECK-LABEL: for function 'divergent_args':
+; CHECK: DIVERGENT ARGUMENTS
+define void @divergent_args(i32 %i32, i1 %i1) {
+ ret void
+}
+
+; CHECK-LABEL: for function 'no_divergent_args_if_inreg':
+; CHECK-NOT: DIVERGENT
```

Comment:
I think i1 values should still be considered divergent; as a separate change I still want to have those be interpreted as lane masks. It would be better to have a separate test function for the two cases. It's fine for now in this patch

Reply:
Ok!